### ISSUE TITLE
Android sets / returns values in a different scale

### ISSUE THREAD

**Issue by nll** [jfversluis/Plugin.Maui.ScreenBrightness#1]
From the Android documentation:

 - [Settings.System#SCREEN_BRIGHTNESS](https://developer.android.com/reference/android/provider/Settings.System#SCREEN_BRIGHTNESS) the return value is between 0 and 255
 - [WindowManager.LayoutParams#screenBrightness](https://developer.android.com/reference/android/view/WindowManager.LayoutParams#screenBrightness) it seems the accepted values are between 0 and 1

Also from my testing the system setting does not reflect the value set on the window layout params, so I suspect the code should be something like this:
```csharp
var context = Platform.CurrentActivity;
if (context == null) {
    return 0;
}

var windowBrightness = window.Attributes.ScreenBrightness;
if (windowBrightness < 0) {
    return Settings.System.GetInt(context.ContentResolver, Settings.System.ScreenBrightness) / 255;
}

return windowBrightness;
```

**Comment by jfversluis** [comment]
Oh good catch! If you want, feel free to open a PR!

**Comment by jfversluis** [comment]
Fixed by #2
